sahar.saharj/hw_5/templating/index.html

```diff
@@ -0,0 +1,45 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <title>Popular Recipes</title>
+        <link rel="stylesheet" href="style.css">
+    </head>
+    <body>
+        <h1>Popular Recipes</h1>
+        <!-- <div class="recipe">
+            <div><span>Name: </span><span>Shakshuka</span></div>
+            <div><span>Serves: </span><span>4</span></div>
+            <div><span>Meal: </span><span>breakfast</span></div>
+        </div>
+        <div class="recipe">
+            <div><span>Name: </span><span>Black Bean Chili</span></div>
+            <div><span>Serves: </span><span>8</span></div>
+            <div><span>Meal: </span><span>dinner</span></div>
+        </div>
+        <div class="recipe">
+            <div><span>Name: </span><span>Classic Peanut Butter Cookies</span></div>
+            <div><span>Serves: </span><span>3 dozen</span></div>
+            <div><span>Meal: </span><span>dessert</span></div>
+        </div> -->
+
+        <script id="recipe-template" type="text/x-handlebars-template">
+            {{#each items}}
+                <div class="recipe">
+                    <div><span>Name: </span>
+                    {{name}}
+                    </div>
+                    <div><span>Serves: </span>
+                    {{serve}}
+                    </div>
+                    <div><span>Meal: </span>
+                    {{meal}}
+                    </div>
+                </div>
+            {{/each}}
```

Comment:
Thank you!
I feel more confident building pages with JS/jQuery, but I'd like to get some experience working with templates too.